tests/test_cli_sources.py

```diff
@@ -91,6 +92,41 @@ def test_cli_add_source_no_validation(mock_request_post, MockResponse):
     )
 
 
+@pytest.mark.usefixtures("token_environ")
```

Comment:
Could you add a test for upload-source without --replace?

Reply:
👍